### DISCUSSION TITLE
Automatic resizing of image

### DISCUSSION THREAD

**Discussion by sclaes** [ImageMagick/ImageMagick#4990] [Help]
Hello,

I'm probably using an old version but Ubuntu 20.04.1 (apt install) doesn't install a newer version.
  Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101

When I display an image (with the display command) and resize the window, the image resizes too. It was possible to disable this by pressing F (fixed size). This doesn't work anymore. Is there any way to disable this automatic resizing?

Any suggestions? TIA

**Comment by urban-warrior** [comment]
If you resize the window quickly, the image won't resize.  If you resize and hold the button, the image does resize.
